xtas/tasks/_polarity.py

```diff
@@ -68,7 +68,7 @@ def train(param_search=False):
                                         strip_accents='unicode'),
                         LogisticRegression(random_state=623, C=5000))
 
-    if param_search:
```

Comment:
Does it make sense to split this? I.e. have two separate model variables and two separate functions that each train, store and load one model?